concourse/pipelines/replicator.py

```diff
@@ -301,9 +303,15 @@ def deploy(self, definition_descriptor):
                 api.unpause_pipeline(pipeline_name=pipeline_name)
             if self.expose_pipelines:
                 api.expose_pipeline(pipeline_name=pipeline_name)
+
+            if response is client.SetPipelineResult.CREATED:
+                deploy_status = DeployStatus.CREATED
+            else:
+                deploy_status = DeployStatus.SUCCEEDED
```

Comment:
please change to elif client.SetPipelineResult.UPDATED + raise  NotImplementedError() else